ccd-definition/CaseField/CareSupervision/tabHeadings.json

```diff
@@ -2,9 +2,9 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "courtBundleTabHeading",
+    "ID": "hearingDocumentsTabHeading",
```

Comment:
not sure if we need to worry about deprecating here too?

Reply:
added to CaseField/CareSupervision/DEPRECATED.json